tests/test_check_swatinit_simulators.py

```diff
@@ -117,48 +120,28 @@ def test_swat_limited_by_ppcwmax_above_contact(simulator, tmp_path):
     qc_frame = run_reservoir_simulator(simulator, model)
     assert np.isclose(qc_frame["PPCWMAX"][0], 3.01)
     qc_vols = qc_volumes(qc_frame)
-    swat_if_ppcwmax = 0.5746147
     if "eclipse" in simulator:
         # for PPCWMAX set to 3.01 Eclipse100 will scale the swatinit value to this:
-        assert qc_frame["QC_FLAG"][0] == __PPCWMAX__
-        assert np.isclose(qc_frame["SWAT"][0], swat_if_ppcwmax)
-        assert np.isclose(qc_frame["PC_SCALING"][0], 3.01 / 3.00)
         actual_pc = 1.4226775
-        assert np.isclose(
-            model.evaluate_pc(swat_if_ppcwmax, scaling=3.01 / 3.00), actual_pc
-        )
-        assert np.isclose(qc_frame["PC"][0], actual_pc)
-
-        assert np.isclose(
-            qc_vols[__PPCWMAX__], (swat_if_ppcwmax - swatinit) * qc_frame["PORV"][0]
-        )
+        swat_if_ppcwmax = 0.5746147
     else:
-        # "flow" does not support PPCWMAX and will scale the Pc curve as much is needed
-        # and will thus reproduce swatinit:
-        assert qc_frame["QC_FLAG"][0] == __PC_SCALED__
-        assert np.isclose(qc_frame["SWAT"][0], swatinit)
-        expected_scaling = 2.107745
-        assert np.isclose(qc_frame["PC_SCALING"][0], expected_scaling)
-
-        actual_pc = 1.405163
-        # The capillary pressure in this cell:
-        assert np.isclose(
-            model.evaluate_pc(swatinit, scaling=expected_scaling), actual_pc
-        )
-        assert np.isclose(qc_frame["PC"][0], actual_pc)
-        # If we had done scaling according to PPCWMAX, we would have gotten
-        # the same result as in Eclipse:
-        assert np.isclose(
-            # Worryingly inaccurate?
-            model.evaluate_sw(actual_pc, scaling=3.01 / 3.00),
-            swat_if_ppcwmax,
-            atol=0.02,
-            # (opm probably uses monotone cubic interpolation in SWOF, but that
-            # should not affect SWOF tables with only two points)
-        )
+        # OPM Flow gives different swat due to different resolution
+        # in number of sections to calculate equilibrium
+        # Ref: email from Torbjørn Skille
```

Comment:
The named reference is of limited use in the source code repository, only the explanation itself is verifiable (it sounds a little bit like FINE_EQUIL being enabled by default?)

Reply:
I'm not sure what is FINE_EQUIL that you are referring.

The explanation from Torbjørn is that:

> Capillary pressure are calculated from densities which are dependent on pressure. The pressure will vary with depth and this is of course not linear. I have not programmed this myself but the depth interval covered with the grid needs to be divided into  a finite number of sample points and capillary pressures are then calculated for these points based on pressure and capillary pressure specified at datum depth.  The number of sampling points is not exposed for users, nightery in Eclipse of OPM flow. This is happening inside the simulator. 

I think he is referring to this code:
https://github.com/OPM/opm-simulators/blob/815bb7a49369a1a765cc5f9836144a8833c5d5aa/ebos/equil/initstateequil.hh#L169 

When he varied these number of points, he found that the Pc from Eclipse is obtained when using 400 sections and OPM is using 2000 sections by default
![image](https://github.com/equinor/subscript/assets/11570927/27ebc0bc-5512-4513-aa7d-5bbc244f92ed)